app/models/starting_child.rb

```diff
@@ -23,7 +23,14 @@ def benefits_end
   # Return the next Monday if start_date is not nil.
   def adjusted_start_date
     return nil if start_date.nil?
-    start_date + (start_date.wday < 1 ? 1 : (1 - start_date.wday) + 7)
+
+    if start_date.wday < 1
+      start_date + 1
+    elsif start_date.wday > 1
+      start_date + ((1 - start_date.wday) + 7)
+    else
+      start_date
+    end
```

Comment:
I believe what you want here is:

if start_date.wday < 1
  start_date + 1.days
elsif start_date.wday > 1
  start_date + ((1 - start_date.wday) + 7).days
else
  start_date
end